tests/test_remotescript.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=redefined-builtin
```

Comment:
Consider creating conftest.py with all fixtures. In this fashion there will be no need to disable **redefined-builtin**